magento/magento2
# Fix > Minicart caching problems

### Description (*)

Fix > Minicart caching problems

### Related Pull Requests

N/A

### Fixed Issues (if relevant)

Fixes #3625

### Manual testing scenarios (*)

1. From Magento Admin go to Catalog - Products, Add a Simple Product
2. Fill in all the required fields, set Price 100 for example, Save the Product
3. Open Store View, and Add the previously created Product to Cart
4. From Magento Admin go to Catalog - Products, and Edit the previously created product, set Price to 90, for example. Save
5. Open Store View, refresh the web page
6. Click on Cart in the top right corner and verify the Price

### Questions or comments

I don't know if my solution is the best but worked, the logic/reasoning is: 

We need to know on all pages that has minicart if the quote.updated_at in the database is different from localStorage (cache), if it's different the minicart cache must be invalidated and reloaded.

### Contribution checklist (*)
